docs/start/getting-started/fragments/vanillajs/data-model.md

```diff
@@ -34,11 +34,12 @@ The `@model` directive lets Amplify know we intend for this type to have data th
 Now that the data is modeled, it's time to create the GraphQL API. From the root of the project, run the following:
 
 ```bash
-amplify add api #accept defaults
+amplify add api
 ```
 
-The default values are highlighted below.
-```bash
+Accept the default values which are highlighted below:
```

Comment:
user studies have shown that people don't read much outside of codeblocks. we should make these either bold or retain them in comments